code/src/components/ShowMovie.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable linebreak-style */
+import React, { useState, useEffect } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+
+export const ShowMovie = () => {
+  const [details, setDetails] = useState([])
+  const navigate = useNavigate()
+  const { id } = useParams()
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=5b17ba21606be74c35e11124051ec659`)
+      .then((res) => res.json())
+      .then((json) => { setDetails(json) })
+      .catch((error) => alert(error, 'error'))
+      .finally(() => console.log('movie-details'))
+  }, [id])
+
+  const goBack = () => {
+    navigate(-1)
+  }
+
+  return (
+    <section className="details-page">
```

Comment:
To display a background as well, you could add this code inside of the section tag for "details-page" 
`style={{ backgroundImage: `url(https://image.tmdb.org/t/p/w1280${details.backdrop_path})` }}`